perl/lib/Utils/scripts/cadd_indel_to_vcf.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/env bash
+
+# Function to show help
+show_help() {
+    echo "Usage: $0 [OPTION]..."
+    echo "Convert a custom file format to VCF."
+    echo "Reads from standard input if '-' is provided as an argument, or use --input to specify a file."
+    echo
+    echo "  -h, --help    Display this help and exit"
+    echo "  --input FILE  Specify the input file to be converted"
+}
+
+# No arguments provided (not even a dash)
+if [ "$#" -eq 0 ]; then
+    show_help
+    exit 1
+fi
+
+# Variables
+input_file=""
+
+# Parse command-line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -h|--help)
+            show_help
+            exit 0
+            ;;
+        --input)
+            if [[ -n $input_file ]]; then
+                echo "Error: Cannot use --input with '-' argument."
+                exit 1
+            fi
+            input_file="$2"
+            shift # Remove argument name from processing
+            shift # Remove argument value from processing
+            ;;
+        -)
+            if [[ -n $input_file ]]; then
+                echo "Error: Cannot use '-' with --input argument."
+                exit 1
+            fi
+            input_file="-" # Indicate that we should read from stdin
+            shift
+            ;;
+        *)
+            # Unknown option
+            echo "Unknown option: $1"
+            show_help
+            exit 1
+            ;;
+    esac
+done
+
+# VCF Header
+cat <<EOF
```

Comment:
I think it would make more sense to include printing the vcf header after confirming the input file below, within the if statement.